docs/src/develop/rest-api/course_api.md

```diff
@@ -235,12 +237,20 @@ _Example response:_
 
 ## Cancelling navigation
 
-To cancel the current course navigation and clear the course data
+To cancel the current course navigation and clear the course data.
 
 ```typescript
 HTTP DELETE 'http://hostname:3000/signalk/v2/api/vessels/self/navigation/course/'
 ```
 
+To clear the current destination and stop NMEA stream input from setting the destination, specify the `force` parameter.
+
+```typescript
+HTTP DELETE 'http://hostname:3000/signalk/v2/api/vessels/self/navigation/course?force=1'
```

Comment:
Imho `true` would be less ambiguous and clearer than `1`.

Reply:
Both are actually supported in the code.
Will update the doc accordingly.